src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Win32.cs

```diff
@@ -186,14 +188,29 @@ private static ProcessModuleCollection GetModules(int processId, bool firstModul
 
                         module.ModuleName = new string(chars, 0, length);
 
-                        length = Interop.Kernel32.GetModuleFileNameEx(processHandle, moduleHandle, chars, chars.Length);
+                        for (; ; )
```

Comment:
Nit: We've standardized on:
```suggestion
                        while (true)
```